src/routes/router.tsx

```diff
@@ -0,0 +1,40 @@
+import App from '@/App';
+import Login from '@/pages/auth/Login';
+
+import { createBrowserRouter } from 'react-router-dom';
+import Protected from './Protected';
+import { Register } from '@/pages/auth/Register';
+import Profile from '@/pages/profile/Profile';
+import NotFound from '@/pages/NotFound';
+import Articles from '@/pages/articles/Articles';
+
+export const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <App />,
+  },
+  {
+    path: '/articles',
+    element: <Articles />,
+  },
+  {
+    path: '/dashboard',
+    element: (
+      <Protected>
+        <Profile />
+      </Protected>
+    ),
+  },
+  {
+    path: '/login',
+    element: <Login />,
```

Comment:
@BardhMustafa can a logged-in user go to Login page?

Reply:
I think /login should be a public route, and then handle the case if user is logged inside Login page

Reply:
I have a Protected Wrapper that checks pages that requires authentication to continue, maybe i can customize that wrapper and use for all pages.  Not only protected ones.

Reply:
@BardhMustafa I think you can improve this logic a bit, here is an example of routes
Public => (no need for wrapper) These routes are not guarded and are accessible from everyone either logged-in or non-logged-in users. (ex. Intro page, 404 page, FAQ, Terms and Conditions etc.)

Protected => (this will need a wrapper) These routes are guarded against logged-in users. (ex. Login page is visible only for non-logged in user, or Forgot Password page should not be visible for user that already are logged-in). This means that routes here are visible only if user is not-logged-in.

Private => (this will need a wrapper) These routes are private and user needs to authenticate in order to visit these pages (ex. Profile, Reset Password and etc.)

You might need to refactor a bit the existing ProtectedWrapper to rename to PrivateWrapper and create another Protected that will be used as mentioned above

Reply:
thanks for this explanation, soon i'll push some changes.